docs/products/postgresql/howto/manage-pool.rst

```diff
@@ -0,0 +1,52 @@
+Manage Connection Pooling
+=========================
+
+Connection Pooling allows you to maintain very large numbers of connections to a database while minimizing the consumption of server resources. Read more about it on :doc:`../concepts/pg-connection-pooling`.
+
+1. Log in to the Aiven web console and select your PostgreSQL service.
+
+   You can connect directly to the PostgreSQL server using the settings listed on the *Overview* page (the *Service URI* address), but this type of connection does not use PGBouncer pooling.
```

Comment:
This is useful information but it definitely isn't step 1! Can we find somewhere else to mention this?